app/services/pubsubhubbub/subscribe_service.rb

```diff
@@ -6,9 +6,9 @@ class Pubsubhubbub::SubscribeService < BaseService
   attr_reader :account, :callback, :secret, :lease_seconds
 
   def call(account, callback, secret, lease_seconds)
-    @account = account
-    @callback = callback
-    @secret = secret
+    @account       = account
+    @callback      = Addressable::URI.parse(callback).normalize.to_s
```

Comment:
Could this stay as a straight assignment, and then add a `normalized_callback_url` (or similar) method below?